test/Fuzzing/fuzzer_test.swift

```diff
@@ -0,0 +1,13 @@
+// RUN: %target-build-swift -parse-as-library -sanitize=fuzzer %s -o %t
+// RUN: not %t -only_ascii=1 -max_len=3 | %FileCheck %s
+// CHECK: Crash!
```

Comment:
@jrose-apple yes, that certainly can be checked. It used to work on Linux, so that's a bug, and I've thought that all bots had compiler-rt?..